common/app/views/fragments/header.scala.html

```diff
@@ -92,7 +92,7 @@
                             data-edition="@{editionId}"
                             href="@getJobUrl(editionId)">
 
-                                <span class="hide-until-tablet">Find a job</span>
+                                <span class="hide-until-tablet">Search jobs</span>
```

Comment:
Does design know about this change? In particular cc @zeftilldeath it might impact your work.

Reply:
Thanks Natalia, I can reflect this on the new nav